### PR TITLE
[DASH-2362] Set 'default' as hosting name causes display default hosting url

### PR DESCRIPTION
**Pull request by Flixow**

### CODE REVIEW

**Comment by Flixow** [comment]
It should be 'default' as prefix

**Comment by Flixow** [comment]
image

File: src/apps/Hosting/HostingDialog.js [hunk 1]
```diff
@@ -19,15 +19,30 @@ const CreateHostingDialog = React.createClass({
...
+          maximum: 256
+        }
       }
+    };
+
+    if (name.toLowerCase() === 'default') {
+      validateObj.name = {
+        inclusion: {
+          within: [],
+          message: "^Name should be another than 'default'"
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingDialog.js]: `can't be a 'default'`

File: src/apps/Hosting/HostingDialog.js [hunk 2]
```diff
@@ -19,15 +19,30 @@ const CreateHostingDialog = React.createClass({
...
+        presence: true
+      },
+      description: {
+        length: {
+          maximum: 256
+        }
       }
+    };
+
+    if (name.toLowerCase() === 'default') {
```
Comment (wpasinski) [inline src/apps/Hosting/HostingDialog.js]: how about exclusion?

https://validatejs.org/#validators-exclusion

**Comment by gh3r** [comment]
@Flixow LGTM

**Pull request merged**